experimental/generation/generator/packages/library/templates/standard/standard.schema

```diff
@@ -59,6 +59,16 @@
     "$templates": [
         "booleanEntity.lg",
         "booleanEntity.lu",
+        "datetimeV2Entity.lg",
+        "emailEntity.lg",
+        "keyPhraseEntity.lg",
+        "personNameEntity.lg",
+        "phonenumberEntity.lg",
+        "urlEntity.lg",
+        "stringEntity.lg",
+        "numberEntity.lg",
+        "integerEntity.lg",
+        "percentageEntity.lg",
         "form-NoneIntent.lu",
```

Comment:
You should not put there here--they should be part of the entity specific templates, i.e. only generated if the entity is used.  For example, the phoneNumberEntity.lg should be pointed to by emailEntity.lg.lg which uses entityLGTemplates which points to emailEntity.en-us.lg and that is where the LG display function should be.  If you turn on the --debug flag you can see where templates get connected.